### ISSUE TITLE
Remove ConfirmViewController.xib

### ISSUE THREAD

**Issue by ppamorim** [AlexLittlejohn/ALCameraViewController#77]
We should use the `AutoLayout` to generate this view.
Why the `xib` is needed for this view? We can create it using Swift code. 

Thanks!

**Comment by AlexLittlejohn** [comment]
I agree with this. However my auto layout knowledge is not enough to achieve the on rotate state that it currently has.
